cmd/snap-confine/snap.h

```diff
@@ -21,5 +21,7 @@
 #include <stdbool.h>
 
 bool verify_security_tag(const char *security_tag);
+bool verify_is_hook_security_tag(const char *security_tag);
```

Comment:
`verify_is_hook_security_tag` feels mis-named. Is it checking if a security tag is a hook security tag or checking if a hook security tag is valid? Please name it more appropriately.